client/src/components/avatar/EntityAvatarDisplay.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from "prop-types"
+import React, { useEffect } from "react"
+
+export const EntityAvatarDisplay = ({ avatar, width, height }) => {
+  const avatarStyle = {
+    width: `${width}px`,
+    height: `${height}px`,
+    maxWidth: "100%",
+    display: "block",
+    margin: "0 auto",
+    marginBottom: "10px"
+  }
```

Comment:
Well this is how the EntityAvatar for person is done. Again, not sure is worth spending the time on it now.

Reply:
Since the way it's done for people is going to change to this one, why not do it now? It is better in terms of React style (fewer re-renderings for example).